pages/gi-bill-comparison-tool/index.md

```diff
@@ -1,11 +0,0 @@
----
-title: GI Bill Comparison Tool
-display_title: GI Bill school comparison
-description: Use our GI Bill Comparison Tool to help you decide which education program and school is best for you. Find out which benefits you’ll get at your chosen school.
-layout: page-react.html
-entryname: gi
-collection: education
-spoke: More resources
-order: 1
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-b6d482de8b79905b2e301852ab93a6e3R5.